samples/matter/light_bulb/README.rst

```diff
@@ -339,6 +339,61 @@ Upgrading the device firmware
 
 To upgrade the device firmware, complete the steps listed for the selected method in the :doc:`matter:nrfconnect_examples_software_update` tutorial in the Matter documentation.
 
+AWS IoT Integration
+===================
+
+The sample can be configured to communicate with AWS IoT Core to control attributes in supported clusters on the device.
+When properly setup, the sample will mirror these attributes in the AWS IoT shadow document.
+This makes it possible to remotely control the device using the `AWS IoT Device Shadow Service`_.
+The supported attributes are onoff and levelcontrol.
+
+To setup an AWS IoT instance and configure the sample, follow these instructions:
+
+1. Follow the AWS IoT library documentation to get the hostname, device ID, and certificates used in the connection: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/aws_iot.html
```

Comment:
Could we reference to `lib_aws_iot` using `ref:` instead of pasting the raw link?